components/com_finder/tmpl/search/default_result.php

```diff
@@ -48,6 +49,22 @@
 	$description = HTMLHelper::_('string.truncate', StringHelper::substr($full_description, $start), $desc_length, true);
 }
 
+$show_image = $this->params->get('show_image', 0) && !empty($this->result->getImage()) ? true : false;
+
+if ($show_image)
+{
+    $resultImageObject = $this->result->getImage();
```

Comment:
```suggestion
	$resultImageObject = $this->result->getImage();
```
CS: Use tabs and not spaced to indent code.